packages/@aws-cdk/aws-autoscaling/lib/lifecycle-hook.ts

```diff
@@ -46,13 +46,15 @@ export interface BasicLifecycleHookProps {
 
   /**
    * The target of the lifecycle hook
+   *
+   * @default - No target.
```

Comment:
When I said "explain it" I actually didn't just mean "explain it to me"... explain it to everyone! Describe in the doc comment what not having a target does.

How are users supposed to decide whether or not it makes sense for them to leave this optional parameter out otherwise?

Answer "because CFN said so" isn't good enough, I'm afraid, because it doesn't lead to any *understanding*, and it doesn't help me make a choice. At this point you're going to have to look at the service documentation for this feature and tease out what it means.

If you can't find that information, that would be a good opportunity to cut a ticket to the docs team (there's a way to do that at the bottom of every page) to call out that missing information.